.github/workflows/dockerimage.yml

```diff
@@ -0,0 +1,31 @@
+name: top trumps mongoDB CI
+
+on:
+  push:
+    branches: [ master, develop, ci]
```

Comment:
You don't need to include ci here.

Reply:
@mingYi-ch I don't have to add anything, I'd suggest you apply the requested change from @kaushik-rohit.